core/templates/dev/head/base.js

```diff
@@ -95,6 +97,11 @@ oppia.controller('Base', [
 
     // If this is nonempty, the whole page goes into 'Loading...' mode.
     $rootScope.loadingMessage = '';
+    // Method to call windowSize on resizing the window
+    $window.onresize = function() {
```

Comment:
No. I was not intending to continuously poll the page's width. If Re calling services is not an issue then this method is totally fine.
Thanks.